paddle/fluid/operators/mul_op.cc

```diff
@@ -114,6 +144,23 @@ class MulOpMaker : public framework::OpProtoAndCheckerMaker {
         )DOC")
         .SetDefault(1)
         .EqualGreaterThan(1);
+    AddAttr<float>("scale_x",
+                   "scale_x to used for int8 input data x."
+                   "Only used with MKL-DNN INT8")
+        .SetDefault(1.0f);
+    AddAttr<std::vector<float>>("scale_y",
+                                "scale_y to used for int8 input data y."
+                                "Only used with MKL-DNN INT8")
+        .SetDefault({1.0f});
+    AddAttr<float>("scale_out",
+                   "scale_out to be used for int8 output data."
+                   "Only used with MKL-DNN INT8")
+        .SetDefault(1.0f);
+    AddAttr<bool>(
+        "force_fp32_output",
+        "(bool, default false) Force quantize kernel output FP32, only "
+        "used in quantized MKL-DNN.")
```

Comment:
Is  `scale_x`, `scale_y`, `scale_out` and `force_fp32_output` all needed?

Reply:
`scale_x`, `scale_y` and `force_fp32_output` are needed for QAT, same as in conv2d. Besides `scale_out` is needed for int8v2.

Reply:
Why int8v2 use the different attribute from QAT?

Reply:
int8v2 will use all the attributes mentioned above. `scale_out` is used to convert s32 output -> s8 .